src/ngx-gallery-preview.component.ts

```diff
@@ -113,6 +115,11 @@ export class NgxGalleryPreviewComponent implements OnChanges {
         }
     }
 
+    zoomChanged() {
```

Comment:
Now here you would get the `event` and could access e.g. the current `ZoomPosition` with `event.after` (or you name the variable `zoomPositions`, so that the code reads more clearly => `zoomPositions.after`).